src/StateSmith/Output/UserConfig/IRenderConfigC.cs

```diff
@@ -33,4 +33,9 @@ public interface IRenderConfigC : IRenderConfig
     /// https://github.com/StateSmith/StateSmith/issues/185
     /// </summary>
     string CEnumDeclarer => "";
+
+    /// <summary>
+    /// Will use &lt;stdbool.h&gt; and bool type instead of int for boolean
+    /// </summary>
+    string UseStdBool => "true";
```

Comment:
We may one day drop support for the old style diagram render config specification, but for now we should support the new item here: https://github.com/StateSmith/StateSmith/blob/556c4612fddd1fab378ce7287623f260ad6813b7/src/StateSmith/SmGraph/RenderConfigVerticesProcessor.cs#L121

Treat like this: https://github.com/StateSmith/StateSmith/blob/556c4612fddd1fab378ce7287623f260ad6813b7/src/StateSmith/SmGraph/RenderConfigVerticesProcessor.cs#L131